doc/api-extensions.md

```diff
@@ -717,7 +717,7 @@ used to track the current mapping for the container.
 
 This effectively gives us:
 
- - `volatile.last\_state.idmap` => On-disk idmap
+ - `volatile.last_state.idmap` => On-disk idmap
```

Comment:
Out of curiosity, do you know why the escaping was in place in the first place? Just want to double check we're not regressing something we don't know about (e.g. parsing scripts that read `api-extensions.md`). I always found it odd too, btw, but never quite investigated.

Reply:
@freeekanayaka the escaping is needed for values that are not inside backticks. I think that is where the confusion has come from. See also https://github.com/lxc/lxd/pull/7825#discussion_r478584302